actstream/managers.py

```diff
@@ -134,40 +134,50 @@ class FollowManager(GFKManager):
     Manager for Follow model.
     """
 
-    def for_object(self, instance):
+    def for_object(self, instance, follow_type=None):
         """
         Filter to a specific instance.
         """
         check(instance)
         content_type = ContentType.objects.get_for_model(instance).pk
-        return self.filter(content_type=content_type, object_id=instance.pk)
+        queryset = self.filter(content_type=content_type, object_id=instance.pk)
+        if follow_type:
```

Comment:
what about if follow_type is not None: ?

Reply:
`follow_type`  supposed to be a string. If it is None, all follow records will be returned. Otherwise records of that follow type will be returned.